docs/system_requirements/ci/aws_codebuild.md

```diff
@@ -0,0 +1,18 @@
+# AWS CodeBuild
```

Comment:
@kiview I think this entire section should live in the testcontainers.com website

Reply:
Thanks @eddumelendez! we should probably take a note on this for an eventual revamp of the websites